### PR TITLE
add the beginnings of a migrations test for the catalog

### PR DESCRIPTION
**Pull request by freben**

### CODE REVIEW

File: plugins/catalog-backend/src/database/migrations.test.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        .into('search');
+
+      await expect(knex('search')).resolves.toEqual(
+        expect.arrayContaining([
+          { entity_id: 'i', key: 'k1', value: 'v1' },
+          { entity_id: 'i', key: 'k2', value: null },
+        ]),
+      );
+
+      await knex.migrate.up({ directory: migrationsDir });
```
Comment (Rugvip) [inline plugins/catalog-backend/src/database/migrations.test.ts:77]: While you're at it I think it makes sense to try to migrate back down as well. At the very least making sure it's possible, assertions optional

**Review by Rugvip — APPROVED** [review]
👌 

Very much a detail, but if we make this a general pattern, do you think it would make sense to always keep it in `src/migrations.test.ts` instead?

**Comment by freben** [comment]
I just put it next to the migrations.ts file that already existed :)

File: plugins/catalog-backend/src/database/migrations.test.ts (hunk 1 above)
Comment (freben) [reply plugins/catalog-backend/src/database/migrations.test.ts]: Ah yeah of course, added!

**Comment by freben** [comment]
but yeah ok, moved it to the root, i think it's a good idea

**Pull request merged**